en/SUMMARY.md

```diff
@@ -197,3 +197,99 @@
+- [Editor.UI.Settings](extension/api/editor-framework/renderer/ui/settings.md)
+- [Editor.Window](extension/api/editor-framework/renderer/window.md)
+- [Editor.Easing](extension/api/editor-framework/share/easing.md)
+- [Editor](extension/api/editor-framework/share/editor.md)
+- [Editor.i18n](extension/api/editor-framework/share/i18n.md)
+- [Editor.IpcListener](extension/api/editor-framework/share/ipc-listener.md)
+- [Editor.JS](extension/api/editor-framework/share/js-utils.md)
+- [Editor.Math](extension/api/editor-framework/share/math.md)
+- [Editor.Selection](extension/api/editor-framework/share/selection.md)
+- [Editor.Undo](extension/api/editor-framework/share/undo.md)
+- [Editor.Utils](extension/api/editor-framework/share/utils.md)
+- [Editor API Reference](extension/api/index.md)
+- [扩展 Property](extension/extends-property.md)
+- [meta 文件参考](extension/reference/meta-reference.md)
+- [ui-asset 参考](extension/reference/ui-asset.md)
+- [ui-box-container 参考](extension/reference/ui-box-container.md)
+- [ui-button 参考](extension/reference/ui-button.md)
+- [ui-button 参考](extension/reference/ui-checkbox.md)
+- [ui-color 参考](extension/reference/ui-color.md)
+- [ui-hint 参考](extension/reference/ui-hint.md)
+- [ui-input 参考](extension/reference/ui-input.md)
+- [ui-loader 参考](extension/reference/ui-loader.md)
+- [ui-markdown 参考](extension/reference/ui-markdown.md)
+- [ui-node 参考](extension/reference/ui-node.md)
+- [ui-num-input 参考](extension/reference/ui-num-input.md)
+- [ui-progress 参考](extension/reference/ui-progress.md)
+- [ui-prop 参考](extension/reference/ui-prop.md)
+- [ui-section 参考](extension/reference/ui-section.md)
+- [ui-select](extension/reference/ui-select.md)
+- [ui-slider 参考](extension/reference/ui-slider.md)
+- [ui-text-area 参考](extension/reference/ui-text-area.md)
+- [publish-in-linux](publish/publish-in-linux.md)
+- [Arc](render/graphics/arc.md)
+- [BezierCurveTo](render/graphics/bezierCurveTo.md)
+- [Circle](render/graphics/circle.md)
+- [Clear](render/graphics/clear.md)
+- [Close](render/graphics/close.md)
+- [Ellipse](render/graphics/ellipse.md)
+- [Fill](render/graphics/fill.md)
+- [fillColor](render/graphics/fillColor.md)
+- [LineCap](render/graphics/lineCap.md)
+- [LineJoin](render/graphics/lineJoin.md)
+- [LineTo](render/graphics/lineTo.md)
+- [LineWidth](render/graphics/lineWidth.md)
+- [MiterLimit](render/graphics/miterLimit.md)
+- [MoveTo](render/graphics/moveTo.md)
+- [QuadraticCurveTo](render/graphics/quadraticCurveTo.md)
+- [Rect](render/graphics/rect.md)
+- [Stroke](render/graphics/stroke.md)
+- [StrokeColor](render/graphics/strokeColor.md)
+- [Recommended coding standards](scripting/reference/coding-standards.md)
+- [JavaScript 标准支持](scripting/reference/javascript-support.md)
+- [Bind node size with SizeProvider](scripting/reference/size-provider.md)
+- [Debug script in the browser](scripting/web-debug-scripts.md)
+- [ui-animation](ui/ui-animation.md)
+
+## CC_HIDE_IN_SUMMARY_END
```

Comment:
Please don't commit section about CC_HIDE_IN_SUMMARY

Reply:
Ok. I will revert back changes in this file.